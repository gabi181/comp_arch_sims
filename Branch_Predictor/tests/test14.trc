32 5 2 0 local_history local_tables not_using_share
0x1AF207E4 T 0x7EB51DC8
0x322014A4 T 0x68784E8C
0x1AF207E4 N 0x7EB51DC8
0x72FD6854 N 0x2EBECD90
0x72FD6854 T 0x2EBECD90
0x30BD9EC8 T 0x4CE0002C
0x45A30480 T 0x285A6DD8
0x1AF207E4 N 0x7EB51DC8
