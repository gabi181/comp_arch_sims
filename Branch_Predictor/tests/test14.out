0x1af207e4 N 0x1af207e8
0x322014a4 N 0x322014a8
0x1af207e4 N 0x1af207e8
0x72fd6854 N 0x72fd6858
0x72fd6854 N 0x72fd6858
0x30bd9ec8 N 0x30bd9ecc
0x45a30480 N 0x45a30484
0x1af207e4 N 0x1af207e8
flush_num: 5, br_num: 8, size: 3264b
